migrations/v6-MIGRATION.md

```diff
@@ -0,0 +1,123 @@
+## v6 API changes
+
+### New
+
+| Name                   |
+|------------------------|
+| `PurchaseOption`       |
+| `GooglePurchaseOption` |
+| `GoogleStoreProduct`   |
+| `Price`                |
+| `PricingPhase`         |
+| `RecurrenceMode`       |
+
+### StoreProduct updates
```

Comment:
should we call out it's an interface now?

Reply:
I added some text